ibis/tests/expr/test_format.py

```diff
@@ -206,16 +207,26 @@ def test_argument_repr_shows_name():
 
 def test_scalar_parameter_formatting():
     value = ibis.param('array<date>')
-    assert str(value) == 'ScalarParameter[array<date>]'
+    assert str(value) == '$(array<date>)'
 
     value = ibis.param('int64').name('my_param')
-    assert str(value) == 'my_param = ScalarParameter[int64]'
+    assert str(value) == 'my_param = $(int64)'
 
 
 def test_same_column_multiple_aliases():
     table = ibis.table([('col', 'int64')], name='t')
     expr = table[table.col.name('fakealias1'), table.col.name('fakealias2')]
     result = repr(expr)
 
-    assert 'fakealias1' in result
-    assert 'fakealias2' in result
+    assert "UnboundTable[r0, name=t]" in result
+    assert "col int64" in result
+    assert "fakealias1 = r0.col :: int64" in result
```

Comment:
Based on the example in the PR's description the double colon shouldn't be present anymore.

Reply:
Yep, let me fix up the tests.

Reply:
I'd prefer to have a simple example like in the description asserted in test_format.py